app/components/Views/Browser/index.js

```diff
@@ -230,7 +230,7 @@ export const Browser = (props) => {
       const existingTabId = route.params?.existingTabId;
       if (newTabUrl && deeplinkTimestamp) {
         // Open url from deeplink.
```

Comment:
Wdyt about updating this JSDoc since ColletibleModal is also [redirecting](https://github.com/MetaMask/metamask-mobile/blob/72339d1045c4a2ad4c107710ec0be73991ee1e03/app/components/UI/CollectibleModal/CollectibleModal.tsx#L95) to here 
```suggestion
        // Open url from deeplink or navigation with timestamp route param.
```